packages/manager/modules/log-to-customer/src/list/controller.js

```diff
@@ -0,0 +1,164 @@
+import moment from 'moment';
+import { RETENTION } from '../constants';
+
+export default class LogToCustomerListCtrl {
+  /* @ngInject */
+  constructor($state, coreURLBuilder, LogToCustomerService, $translate, $q) {
+    this.$state = $state;
+    this.coreURLBuilder = coreURLBuilder;
+    this.LogToCustomerService = LogToCustomerService;
+    this.$translate = $translate;
+    this.$q = $q;
+  }
+
+  $onInit() {
+    this.loading = false;
+    this.accountList = [];
+    this.streams = [];
+    this.streamSubscriptions = {};
+    this.LDPHomePageLink = null;
+    this.selectedAccount = null;
+
+    this.LogToCustomerService.getIcebergLogAccounts().then((data) => {
+      this.accountList = data.map((item) => {
+        return {
+          ...item,
+          formattedName: item.displayName || item.serviceName,
+        };
+      });
+
+      this.selectedAccount =
+        this.accountList.length > 0 ? this.accountList[0] : null;
+      this.getStreamData();
+    });
+  }
+
+  getStreamData() {
+    this.$q
+      .all([this.getSubscribedStreams(), this.getLogAccountStreams()])
+      .then(() => {
+        this.loading = false;
+      });
+  }
+
+  getSubscribedStreams() {
+    this.streamSubscriptions = {};
+    return this.LogToCustomerService.icebergQuery(this.logSubscriptionUrl).then(
+      (data) => {
+        data.forEach((item) => {
+          this.streamSubscriptions[item.streamId] = { ...item };
+        });
+      },
+    );
+  }
+
+  createLogSubscription(streamId) {
+    this.loading = streamId;
+    this.resetErrorMessage();
+    this.LogToCustomerService.post(this.logSubscriptionUrl, {
+      streamId,
+      kind: this.kind,
+    })
+      .then(({ data }) => {
+        this.LogToCustomerService.pollOperation(
+          this.selectedAccount.serviceName,
+          data,
+        ).then(() => this.getStreamData());
+      })
+      .catch(({ data }) => {
+        this.errorMessage = data.message;
+        this.loading = false;
+      });
+  }
+
+  resetErrorMessage() {
+    this.errorMessage = null;
+  }
+
+  deleteLogSubscription(streamId) {
+    this.loading = streamId;
+    this.resetErrorMessage();
+    const { subscriptionId } = this.streamSubscriptions[streamId];
+
+    this.LogToCustomerService.delete(
+      `${this.logSubscriptionUrl}/${subscriptionId}`,
+      {},
+    )
+      .then(({ data }) => {
+        this.LogToCustomerService.pollOperation(
+          this.selectedAccount.serviceName,
+          data,
+        ).then(() => this.getStreamData());
+      })
+      .catch(({ data }) => {
+        this.errorMessage = data.message;
+        this.loading = false;
+      });
+  }
+
+  getSubscriptionLink(streamId) {
+    return this.coreURLBuilder.buildURL(
+      'dedicated',
+      '#/dbaas/logs/:serviceName/streams/:streamId/subscriptions',
+      {
+        serviceName: this.selectedAccount.serviceName,
+        streamId,
+      },
+    );
```

Comment:
you can use template string directly
```suggestion
    return this.coreURLBuilder.buildURL(
      'dedicated',
      `#/dbaas/logs/${this.selectedAccount.serviceName}/streams/${streamId}/subscriptions`,
    );
```